Typo: delete duplicated test case

MakDon opened the pull request:
<!--
Thanks for your contribution!
Please read this comment in its entirety. It's quite important.

# Pull Request title

It should be in the following format:

```
bpo-NNNN: Summary of the changes made
```

Where: bpo-NNNN refers to the issue number in the https://bugs.python.org.

Most PRs will require an issue number. Trivial changes, like fixing a typo, do not need an issue.

# Backport Pull Request title

If this is a backport PR (PR made against branches other than `master`),
please ensure that the PR title is in the following format:

```
[X.Y] <title from the original PR> (GH-NNNN)
```

Where: [X.Y] is the branch name, e.g. [3.6].

GH-NNNN refers to the PR number from `master`.

-->
Delete a duplicated test case in re_tests.
In the original version, line 109 seems to be the same with 112, which means line 112 is duplicated.
According to "Trivial changes, like fixing a typo, do not need an issue.", no issue was created in the https://bugs.python.org.

MakDon committed:
delete duplicated test case

tirkarthi reviewed (approved):
LGTM. commit where tests were changed : bd48d27944453ad83d3ce37b2c867fa0d59a1c15 . 

cc: @serhiy-storchaka

MakDon committed:
Update re_tests.py

edit line 112 .
Before:
('(?s)a.b', 'a\nb', SUCCEED, 'found', 'a\nb'),
After:
('(?s)a.b', 'a\rb', SUCCEED, 'found', 'a\rb'),
